chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -15,9 +15,42 @@ import chisel3.internal.sourceinfo._
   * of) other Data objects.
   */
 sealed abstract class Aggregate extends Data {
+  protected def childLitArgs = getElements.map(_.litArg).foldLeft[Option[Seq[LitArg]]](Some(Seq[LitArg]())) {
+    case (Some(elems), Some(lit)) => Some(elems :+ lit)
+    case _                        => None
+  }
+  override def litArg = {
+    // TODO[grebe] make a lit for Aggregates
+    childLitArgs.map { x => ULit(0, 1.W) }
```

Comment:
Things should fail noisily if this is an unexpected case instead of propagating bad values